### PR TITLE
Pull Request: Feature/race lap time

### PR DESCRIPTION
**Pull request by roboshoes**
I wanted to pull that into **development** because even if it is not styled, the requested feature is fully implemented and tested.

Also - should someone happen to start working on highscores he might want to have that feature included. Therefore, I (and hopefully @tombu) think it should be in there.

_PS: I am aware of minor syntactic issues like inline style. While this is beeing reviewd I keep working on this._

### CODE REVIEW

File: spec/javascripts/unit/slotcars/play/controllers/game_controller_spec.js.coffee [hunk 1]
```diff
@@ -294,7 +299,18 @@ describe 'slotcars.play.controllers.GameController (unit)', ->
       @gameController.restartGame()
       
       (expect @gameController.get 'isCountdownVisible').toBe true
-
+      
+    it 'sholud unset the flag wether the race is finished', ->
```
Comment (DominikGuzei) [inline spec/javascripts/unit/slotcars/play/controllers/game_controller_spec.js.coffee]: misspelled should (sholud)

Comment (roboshoes) [inline spec/javascripts/unit/slotcars/play/controllers/game_controller_spec.js.coffee]: **sholud** is a good friend of mine. His full name is **ebrahim sholud mediram** and he worked with me on this request. It's his job to set and unset flags.

**Comment by roboshoes** [comment]
It should tell what time for what lap:

_lap 1: 1:34:12_
_lap 2: 1:34:12_

**Comment by tombu** [comment]
I agree to merge it into development. At the moment there is no real design for it so we should soon think about how this overlay should look like.

**But: In my opinion the acception on pivotal tracker should wait until the design is implemented.**

**Comment by tombu** [comment]
Why should the lap number be a chore? It should be a part of this pull request!

**Comment by roboshoes** [comment]
Makes sense. But we might think about that the design will come within a another feature where a lot of things will be redesigned. Anyways: I'm fine with leaving it open or closing it.

File: spec/javascripts/unit/slotcars/play/views/result_view_spec.js.coffee [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    @resultView.onLapTimesChange = sinon.spy()
+    @gameControllerMock.set 'lapTimes', [12, 12]
+    
+    (expect @resultView.onLapTimesChange).toHaveBeenCalled()
+  
+  # it 'should listen to the controller wether the race is over or not', ->
+  #   # @resultView.onRaceStatusChange = sinon.spy()
+  #   # @gameControllerMock.set 'isRaceFinished', true
+
+  #   (expect @resultView.onRaceStatusChange).toHaveBeenCalled()
```
Comment (tombu) [inline spec/javascripts/unit/slotcars/play/views/result_view_spec.js.coffee]: Remove or redo uncommenting. Is this test still needed?

Comment (DominikGuzei) [inline spec/javascripts/unit/slotcars/play/views/result_view_spec.js.coffee]: good catch!

Comment (roboshoes) [inline spec/javascripts/unit/slotcars/play/views/result_view_spec.js.coffee]: Sorry ... that was one of your test that was already commented out. But you are right - I need to remove it. My bad!

**Pull request merged**